tests/sanity/tests/collaborative/issues.spec.ts

```diff
@@ -64,4 +66,91 @@ test.describe('Collaborative test for issue', () => {
       estimation: '2h'
     })
   })
+
+  test('Issues status can be changed by another users', async ({ page, browser }) => {
+    const issue: NewIssue = {
+      title: 'Issues status can be changed by another users',
+      description: 'Collaborative test for issue'
+    }
+
+    // open second page
+    const userSecondPage = await getSecondPage(browser)
+    await (await userSecondPage.goto(`${PlatformURI}/workbench/sanity-ws/tracker/`))?.finished()
+    // const leftSideMenuPageSecond = new LeftSideMenuPage(userSecondPage)
+    // await leftSideMenuPageSecond.buttonTracker.click()
+
+    const issuesPageSecond = new IssuesPage(userSecondPage)
+    await issuesPageSecond.linkSidebarAll.click()
+    await issuesPageSecond.modelSelectorAll.click()
+
+    // change status
+    await (await page.goto(`${PlatformURI}/workbench/sanity-ws/tracker/`))?.finished()
+    const issuesPage = new IssuesPage(page)
+    await issuesPage.linkSidebarAll.click()
+    await issuesPage.modelSelectorBacklog.click()
+    await issuesPage.searchIssueByName(issue.title)
+    await issuesPage.openIssueByName(issue.title)
+
+    const issuesDetailsPage = new IssuesDetailsPage(page)
+    await issuesDetailsPage.editIssue({ status: 'In Progress' })
+
+    // check by another user
+    await issuesPageSecond.modelSelectorBacklog.click()
+    // not active for another user
+    await issuesPageSecond.checkIssueNotExist(issue.title)
+
+    await issuesPageSecond.modelSelectorActive.click()
+    await issuesPageSecond.searchIssueByName(issue.title)
+    await issuesPageSecond.openIssueByName(issue.title)
+
+    const issuesDetailsPageSecond = new IssuesDetailsPage(userSecondPage)
+    await issuesDetailsPageSecond.checkIssue({
+      ...issue,
+      status: 'In Progress'
+    })
+  })
+
+  test('First user change assignee, second user should see assigned issue', async ({ page, browser }) => {
+    const newAssignee: string = 'Dirak Kainin'
+    const issue: NewIssue = {
+      title: 'First user change assignee, second user should see assigned issue',
+      description: 'Issue for collaborative test'
+    }
+
+    // open second page
+    const userSecondPage = await getSecondPage(browser)
+    await (await userSecondPage.goto(`${PlatformURI}/workbench/sanity-ws/tracker/`))?.finished()
+
+    // change assignee by first user
```

Comment:
I wonder, can we use steps instead of comments in code for better test case structuring and reporting?
https://playwright.dev/docs/api/class-test#test-step

Reply:
great point. checking